### ISSUE TITLE
Can I install without using docker

### ISSUE THREAD

**Issue by by123456by** [SquadMS/Squad-MS#85]
I want to install on a Windows server. I can't use Docker. Can I install without Docker

**Comment by bumbummen99** [comment]
Yes you can install this without docker, you will most likely have to adjust configuration paths (if there are any) to be Windows like as well as revisit your `.env` to use database / services auth information for your SQL, Redis & SquadMS RCON Worker running on the Windows Server - that does mean you will be setting up mentioned services yourself in a working state refering to their documentation. Also please note i did not test this with MSSQL, only MySQL is tested. 
Id simply start by running it and fixing error after error, most of the time the error message gives an indication on what is going wrong. 

That said, i would highly advise to **not** use Windows (Server) for any PHP development. While a lot of things have cleared up over the PHP7 version cycle PHP on Windows is still not supported very well. PHP on Windows you have to make sure your PHP build is threadsafe and 64 bit, also you are going to have to build all PHP extensions yourself from source - back when i still used Windows there were no binaries provided for those. 

Usually Laravel is being developed locally using Laravel Sail, Homestead or similar solutions to abstract the environment using containers or vm because web and PHP development historically is depencency hell. 

Please keep in mind that right now there is no productive installation guide for SquadMS as it is in early development state right now so there is also no tagged version yet - installing this publicly is not supported until there is a tagged major release, so from version >=1.x.x onwards.
